control_panel_api/pagination.py

```diff
@@ -1,5 +1,43 @@
-from rest_framework.pagination import PageNumberPagination
+from rest_framework.pagination import PageNumberPagination, _positive_int
```

Comment:
Not sure you should import a `_` prefixed function - that implies it is not a part of the public API and could change without notice. I've been burned by this in the past.